R/utils-html.R

```diff
@@ -204,4 +204,30 @@ nullMask <- function(val) {
     result <- ""
   }
   return(result)
+}
+
+#' @title timeFormatting
+#' @description Formats date to passed-in format mask, and time to "(UTC [offset] )"
+#' @param timeVals String with format of "YYYY-MM-DDTHH:MM:SS.SSS-UTC offset".
+#' @param dateFormatMask String with preferred output date format
+#' @return list with date in first position, time in second position.
+timeFormatting <- function(timeVals, dateFormatMask){
+  if(!isEmpty(timeVals)) {
+    dateTime <- (strsplit(timeVals, split="[T]"))
+    dateFormat <- strftime(dateTime[[1]][1], dateFormatMask)
+    
+    #Break apart, format dates/times, put back together.
+    timeFormatting <- sapply(dateTime[[1]][2], function(s) {
+      m <- regexec("([^-+]+)([+-].*)", s)
```

Comment:
You mean the regexec?  Not sure exactly, but the end result is that it breaks the date/time string into a date string and a time string.